arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Either.kt

```diff
@@ -2294,7 +2294,7 @@ public operator fun <A : Comparable<A>, B : Comparable<B>> Either<A, B>.compareT
 
 @Deprecated(
   RedundantAPI + "Prefer zipOrAccumulate",
-  ReplaceWith("Either.zipOrAccumulate({ a, bb -> SGA.run { a.combine(bb) }  }, this, b) { a, bb -> SGB.run { a.combine(bb) } }")
+  ReplaceWith("Either.zipOrAccumulate<A, B, B, B>({ a:A, bb:A -> a.plus(bb) }, this, b) { a:B, bb:B -> a.plus(bb) }")
```

Comment:
Was the `SGA.run  { a.combine(bb) }` here broken? If yes, then I think this is an improvement but we should probably just use `a + b` instead of `a.plus(b)` or is that also broken? 😅

Perhaps we should also add in the description that for custom types, `plus` needs to be replaced with a custom `combine` function?

Reply:
`+` works fine! Thanks @nomisRev . See [`6332005` (#2973)](https://github.com/arrow-kt/arrow/pull/2973/commits/6332005895eb1e90ac788faaf9ffb60b1d0eae41)